components/library/jxrlib/Makefile

```diff
@@ -14,12 +14,13 @@
 #
 
 BUILD_STYLE=	justmake
-
+JAVA_VERSION= 8
+JAVA_HOME= /usr/jdk/openjdk8
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		jxrlib
 COMPONENT_VERSION=	0.2.4
-COMPONENT_REVISION=	1
+COMPONENT_REVISION=	2
 COMPONENT_SUMMARY=	Decode and encode JPEG-XR data from Java
```

Comment:
You don't need to change COMPONENT_REVISION because the last changes haven't been published because of the broken build recipe.